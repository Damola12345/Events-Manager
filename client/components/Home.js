import React from 'react';
import { Link } from 'react-router-dom';
import Header from './commons/Header';
import Footer from './commons/Footer';

/**
 *
 * @class Home
 * @extends {React.Component}
 */
class Home extends React.Component {
  /**
   * @returns {object} component
   * @memberof Home
   */
  render() {
    return (
      <div>
        <Header
          links={{
            centers: 'centers',
            register: 'register',
            login: 'login'
          }}
        />

        <div>
          <section id="introduction">
            <div className="overlay">
              <div className="container">
                <div className="row align-items-center fill-viewport">
                  <div className="col-12">
                    <h1>Party like there is no tomorrow!</h1>
                    <p>Do you plan to hold your dream event, we have centers that can meet your needs.</p>
                    <Link to="/centers" className="btn btn-success">
                      <b>Book a center</b>
                    </Link>
                  </div>
                </div>
              </div>
            </div>
          </section>
        </div>

        <Footer />
      </div>
    );
  }
}

export default Home;
