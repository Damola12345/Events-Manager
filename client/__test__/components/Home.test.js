import React from 'react';
import { shallow } from 'enzyme';
import Home from '../../components/Home';

describe('Home', () => {
  const wrapper = shallow(<Home />);
  it('successfully rendered', () => {
    expect(wrapper).toMatchSnapshot();
  });
});

