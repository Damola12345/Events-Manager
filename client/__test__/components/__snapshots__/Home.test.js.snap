// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home successfully rendered 1`] = `
<div>
  <Connect(Header)
    links={
      Object {
        "centers": "centers",
        "login": "login",
        "register": "register",
      }
    }
  />
  <div>
    <section
      id="introduction"
    >
      <div
        className="overlay"
      >
        <div
          className="container"
        >
          <div
            className="row align-items-center fill-viewport"
          >
            <div
              className="col-12"
            >
              <h1>
                Party like there is no tomorrow!
              </h1>
              <p>
                Do you plan to hold your dream event, we have centers that can meet your needs.
              </p>
              <Link
                className="btn btn-success"
                id="cta-btn"
                replace={false}
                to="/centers"
              >
                <b>
                  Book a center
                </b>
              </Link>
            </div>
          </div>
        </div>
      </div>
    </section>
  </div>
  <Footer
    id="Footer"
  />
</div>
`;
